packages/dashboard/src/components/Wizard/PanelWizard.styles.ts

```diff
@@ -0,0 +1,45 @@
+import { IPanelStyles, ITheme } from 'office-ui-fabric-react';
+import { IWizardStyles } from './Wizard.types';
+import { IPanelWizardStyles, IPanelWizardStyleProps } from './PanelWizard.types';
+
+export const defaultPanelStyleSet = (theme: ITheme): Partial<IPanelStyles> => {
+  return {
+    content: {
+      height: '100%',
+      padding: 0
+    },
+    commands: {
+      display: 'flex',
+      flexDirection: 'row',
+      borderBottom: `1px solid ${theme.semanticColors.bodyDivider}`
+    },
+    scrollableContent: {
+      overflowY: 'hidden'
+    }
+  };
+};
+
+export const defaultWizardStyleSet = (theme: ITheme): Partial<IWizardStyles> => {
+  return {
+    wizardContentNavContainer: {
+      height: '100%'
+    },
+    subwayNavSection: {
+      borderRight: `1px solid ${theme.semanticColors.bodyDivider}`
+    }
+  };
+};
+export const getPanelWizardStyles = (props: IPanelWizardStyleProps): IPanelWizardStyles => {
+  return {
+    footerContainer: {
+      borderTop: `1px solid ${props.theme.semanticColors.bodyDivider}`,
+      padding: `17px 48px 15px ${304 + 48}px`, // 304 is to offset width of subway nav
+      display: 'flex',
+      flexDirection: 'row'
+    },
+    titleElementContainer: {
+      flex: 1,
```

Comment:
Be careful with this shorthand. I believe IE11 interprets this differently than modern browsers. Would be good to confirm it's not a problem. It might be "auto" that it has an issue with but just check to be sure.

Reply:
It should translate into flex-grow: 1 I believe. I will do it in more conventional way just to be safe.

Reply:
Looks like merge-styles automatically expanded it to "1 1 0%" but I made it explicit anyway for clarity.